packages/integration-tests/projects/suite-web/tests/suite/database-migration.test.ts

```diff
@@ -1,46 +1,139 @@
 // @group:migrations
 // @retry=2
 
+import { rerouteMetadataToMockProvider, stubOpen } from '../../stubs/metadata';
+
 describe('Database migration', () => {
     beforeEach(() => {});
 
-    Cypress.env('TEST_URLS').forEach((testUrl: string, index: number) => {
+    const testUrls = ['/release/22.7/web', '/develop/web'];
```

Comment:
I'd keep `Cypress.env('TEST_URLS')` in place in the end. If you want to make local docker/non-docker way of running it more user friendly, we should address this separately.